vault-setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#  ----------------------------------------------------------------------------------
+#  vault-setup.sh    version 1.0 created July 18, 2018
+#
+#  @author:  Tony Espy, Canonical
+#  @email:   espy@canonical.com
+#
+#  Copyright (c) 2018, Canonical Ltd
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  ----------------------------------------------------------------------------------
+
+# Variables and parameters
+echo ">> Setup _VAULT_DIR and fix permissions"
+_VAULT_DIR=${_VAULT_DIR:-/vault}
+
+./pki-setup.sh pki-setup-config-vault.env
```

Comment:
Same comment as above, can we put `pki-setup-config-vault.env` behind an environment variable?

Reply:
If we already control the directory the file is read from, why should the filename make a difference? The snap could copy the .env files to $SNAP_DATA (or some subdir under it) where they could be modified if need be? Then the snaps just needs to set _VAULT_DIR to $SNAP_DATA.